lib/portlayer/exec/base.go

```diff
@@ -331,14 +331,14 @@ func (c *containerBase) poweroff(ctx context.Context) error {
 	})
 
 	if err != nil {
-
 		// It is possible the VM has finally shutdown in between, ignore the error in that case
 		if terr, ok := err.(task.Error); ok {
+			waitForStableOff := false
 			switch terr := terr.Fault().(type) {
 			case *types.InvalidPowerState:
 				if terr.ExistingState == types.VirtualMachinePowerStatePoweredOff {
 					log.Warnf("power off %s task skipped (state was already %s)", c.ExecConfig.ID, terr.ExistingState)
-					return nil
+					waitForStableOff = true
```

Comment:
Am wondering if it's fine to return true here directly, instead of wait another few seconds?
Is there any possible side effect for that?